packages/suite/src/views/view-only/ViewOnlyPromoContent.tsx

```diff
@@ -126,6 +146,18 @@ const ChartText = styled.div`
 const ButtonsContainer = styled.div`
     display: flex;
     gap: ${spacingsPx.sm};
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        flex-flow: row wrap;
+    }
+`;
+
+const ViewOnlyButton = styled(Button)`
+    width: 50%;
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        width: 100%;
+    }
```

Comment:
I'm thinking could we avoid overriding Button component? 🙏 

I think it should be possible with following changes:
- `ButtonsContainer` should contain media queries logic and it should change `flex-direction`
- `Button` component should have `isFullWidth` prop which is the same like `width: 100%`

WDYT?

Reply:
Hmm, `ButtonsContainer` changes - hmm, ok, but without adjusting `Button` itself here I'm not sure if we can achieve the expected look easily. With `isFullWidth` the button takes too much space even for big screens. We'd still need to specify its width and when we do this, using `isFullWidth` becomes unnecessary, so I've removed that. Additionally it doesn't work as expected with css flex props.